tests/run-pass/concurrency/barrier.rs

```diff
@@ -0,0 +1,30 @@
+// ignore-windows: Concurrency on Windows is not supported yet.
+
+use std::sync::{Arc, Barrier};
+use std::thread;
+
+// Check if Rust barriers are working.
```

Comment:
This looks like a duplicate of the existing barrier test in `sync.rs`?

Reply:
Oops it is. I was investigating why `println!` is causing data race so I extracted it out into its own file. Forgot where it came from a couple months later. It's now removed